google/cloud/pubsub/internal/default_pull_ack_handler.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Google LLC
+// Copyright 2023 Google LLC
```

Comment:
nit: I am not sure we /need/ to update the copyright year here.  I think you can, but it may be worthwhile to revisit go/releasing.